docs/FAQ.md

```diff
@@ -0,0 +1,35 @@
+# FAQ - Frequently Asked Questions
+
+
+## My import from Zotero fails, throws errors or behaves unexpectedly
+There can be multiple reason for imports to fail.
+
+- Please verify that your Bibliography Style is set and available in Zotero, too.
+- Please check the [Obsidian Error Console](https://forum.obsidian.md/t/how-to-access-the-console/16703)
+- Please double check your template to see if there is some typo.
+- Use the plugins Data Explorer and check the raw data received from Zotero. This data is used then by the template to extract & format the information.
+
+
+## What are some example templates & steps on how to setup this plugin?
+- Step by step guide: https://publish.obsidian.md/history-notes/01+Notetaking+for+Historians
+- Some template examples: https://forum.obsidian.md/t/zotero-desktop-connector-import-templates/36310
+
+
+## Do I need to extract annotations in Zotero?
+In the past you had to extract annotations in Zotero into its own note.
+
+When using templating, you can extract the annotation directly from Zotero. But this requires setting up a good template that works for you.
+
+The easiest way is to extract annotations and use the "Insert notes into current document". This requires that you export the annotations in Zotero first. Example import:
+
+<img src="https://raw.githubusercontent.com/mgmeyers/obsidian-zotero-integration/main/screenshots/demo.gif" alt="A short gif demonstraiting importing notes form Zotero into the current file">
+
+For more advanced imports & formating, you can build your own templates.
+
+
+## Unanswered questions
+- Image imports
+    - Can the plugin import images from the Zotero annotation?
+    - What to check when image import fails
```

Comment:
Developer console and the image output path / base name:

<img width="535" alt="Screen Shot 2022-10-09 at 9 28 33 AM" src="https://user-images.githubusercontent.com/2694747/194768314-bde0f001-e571-4d57-a225-00d1776a00fc.png">